PKGBUILDS/pine64/acpi-kbd-autolock-pinetab2/toggle-keyboard.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+ACTION=bind
+MESSAGE=on
+if [[ "$3" = "close" ]]
```

Comment:
If we had a command (let's just call it `lid-closed` for example) that detected whether the keyboard's current state is flipped shut, then we could call this script on boot with the following modification:

```suggestion
if [[ "$3" = "close" ]] || lid-closed
```